api/src/main/resources/workbench-api.yaml

```diff
@@ -5254,24 +5254,124 @@ definitions:
     - Deleting
     - Deleted
     - Unknown
+  RuntimeConfigurationType:
```

Comment:
I'd just leave this enum out for now, can wait until the follow-up PR which looks at the runtime labels.